### ISSUE TITLE
CategoryAttributeTests.Ctor_Default failed on netfx in CI

### ISSUE THREAD

**Issue by stephentoub** [dotnet/corefx#24233]
https://ci3.dot.net/job/dotnet_corefx/job/master/job/windows-TGroup_netfx+CGroup_Release+AGroup_x86+TestOuter_false_prtest/3615/consoleText
```
    Discovering: System.ComponentModel.Primitives.Tests
    Discovered:  System.ComponentModel.Primitives.Tests
    Starting:    System.ComponentModel.Primitives.Tests
      System.ComponentModel.Tests.CategoryAttributeTests.Ctor_Default [FAIL]
        Assert.True() Failure
        Expected: True
        Actual:   False
        Stack Trace:
          D:\j\workspace\windows-TGrou---2a8f9c29\src\System.ComponentModel.Primitives\tests\CategoryAttributeTests.cs(18,0): at System.ComponentModel.Tests.CategoryAttributeTests.Ctor_Default()
```

**Comment by danmosemsft** [comment]
Odd. I don't repro this locally, and nothing seems to have changed recently.

**Comment by stephentoub** [comment]
> Odd

It's most likely a race condition.

**Comment by danmosemsft** [comment]
Ah, I already fixed in 013e8523390a41deb3eb8c803b3822f5bc1cc368

**Comment by danmosemsft** [comment]
@maryamariyan can you please disable this test for desktop framework, by adding this attribute to the test method:
`[SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)] // NetFX does not have fix for #21369`

**Comment by danmosemsft** [comment]
[sincey ou're the owner of component model :)]
